src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarAcquireable.tsx

```diff
@@ -0,0 +1,69 @@
+import { Box, Button, Serif } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import { ArtworkSidebarAcquireable_artwork } from "__generated__/ArtworkSidebarAcquireable_artwork.graphql"
+export interface ArtworkSidebarAcquireableProps {
+  artwork: ArtworkSidebarAcquireable_artwork
+}
+
+export const ArtworkSidebarAcquireable: React.SFC<
+  ArtworkSidebarAcquireableProps
+> = props => {
+  const { artwork } = props
+  return (
+    <>
+      <Box pb={2}>
+        {artwork.is_sold ? (
+          <Serif size="5t" weight="semibold">
+            Sold
+          </Serif>
+        ) : (
+          <>
+            <Serif size="5t" weight="semibold">
+              {artwork.price}
```

Comment:
Price is a very low level field on artwork. Definitely preferable to use `sale_message` since it will incorporate bunch of fields in a proper way in MP.